src/metabase/analytics/prometheus.clj

```diff
@@ -219,7 +219,11 @@
                        {:description "Number of entries indexed for search"
                         :labels      [:model]})
    (prometheus/counter :metabase-search/index-ms
-                       {:description "Total number of ms indexing took"})])
+                       {:description "Total number of ms indexing took"})
+   (prometheus/counter :metabase-search/response-ok
+                       {:description "Number of successful search requests."})
+   (prometheus/counter :metabase-search/response-error
```

Comment:
wdyt about `:metabase-search/response` and a label for ok/error? IDK what's more useful though :\

Reply:
I don't like this scheme much, but I wanted to keep consistent with the existing metrics.